iOS/WAStickersThirdParty/AllStickerPacksViewController.swift

```diff
@@ -39,78 +39,125 @@ class AllStickerPacksViewController: UIViewController, UITableViewDataSource, UI
         super.viewDidAppear(animated)
         if needsFetchStickerPacks {
             needsFetchStickerPacks = false
-            fetchStickerPacks()
+            //fetchStickerPacks()
+            fetchStickersOnline()
+            //testParseData()
         }
     }
+    
+    //MARK: Helper
+    //Test json
+    func testParseData () {
+        self.navigationController?.navigationBar.alpha = 1.0;
+
+        guard let path = Bundle.main.path(forResource: "sticker_packs", ofType: "wasticker")  else {
+            return
+        }
+        
+        do {
+            let jsonData = try Data(contentsOf: URL(fileURLWithPath: path), options: Data.ReadingOptions.alwaysMapped)
+            let jsonDecoder = JSONDecoder()
+            let waStickerReponse = try jsonDecoder.decode(WAStickerPackResponse.self, from: jsonData)
+            if let stickerPacks = waStickerReponse.stickerPacks {
+                if stickerPacks.count > 1 {
+                    self.stickerPacks = stickerPacks
+                    self.stickerPacksTableView.reloadData()
+                } else {
+                    self.show(stickerPack: stickerPacks[0], animated: false)
+                }
+            } else {
+                fatalError("No Stickers not found.")
+                
+            }
+            
+            
+        }
+        catch {
+            fatalError("sticker_packs.wasticker not found.")
+        }
+        
+    }
+
+
 
-    private func fetchStickerPacks() {
+//    private func fetchStickerPacks() {
```

Comment:
I think if this gets merged into the master branch, we should at least have both and allow developers to choose which route they want to provide their sticker packs.